browser/ui/views/tabs/brave_tab_container.cc

```diff
@@ -297,11 +297,29 @@ void BraveTabContainer::PaintBoundingBoxForTiles(
 void BraveTabContainer::PaintBoundingBoxForTile(
     gfx::Canvas& canvas,
     const SplitViewBrowserData::Tile& tile) {
+  if (!GetTabCount()) {
+    return;
+  }
+
+  // Note that GetTabAtModelIndex() and IsValidModelIndex() is actually using
+  // "ViewModel" index. See TabContainerImpl::GetTabAtModelIndex(), and
+  // implementations in compound_tab_container.cc implementation. Thus, we need
+  // to add pinned tab count. Also NumPinnedTabsInModel() returns 0 when
```

Comment:
```suggestion
  // to add pinned tab count. Also NumPinnedTabsInModel() returns 0 when ?
```

Reply:
The line can be removed, as we don't use TabContainerImpl::NumPinnedTabs() anymore. Thanks for spotting.